ragna/deploy/_ui/central_view.py

```diff
@@ -429,50 +420,11 @@ def chat_interface(self):
             widgets=[
                 pn.widgets.TextInput(
                     placeholder="Ask a question about the documents",
-                    stylesheets=ui.stylesheets(
-                        (
-                            ":host input[type='text']",
-                            {
-                                "border": "none !important",
-                                "box-shadow": "0px 0px 6px 0px rgba(0, 0, 0, 0.2)",
-                                "padding": "10px 10px 10px 15px",
-                            },
-                        ),
-                        (
-                            ":host input[type='text']:focus",
-                            {
-                                "box-shadow": "0px 0px 8px 0px rgba(0, 0, 0, 0.3)",
-                            },
-                        ),
-                    ),
+                    # css_classes is forced to chat-interface-input-widget by Panel
```

Comment:
Should we report this? Can't we add an additional CSS class with whatever name we want?

Reply:
yes, that's worth reporting to Panel.
if we define the input widget of the chat interface like this : 
```python
# ...
widgets=[
                pn.widgets.TextInput(
                    placeholder="Ask a question about the documents",
                    css_classes=["my_custom_class"]
                )
            ],
# ...
```
The resulting input widget has its css classes forced to `chat-interface-input-widget` instead of adding it.
it's not a problem in our case, but could be for other people trying to style the ChatInterface.

Reply:
See #379.

Reply:
Let's do it in a follow-up PR @pierrotsmnrd